src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,36 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder salaryInfo = new StringBuilder();
+        int salary = 0;
+        for (String name : names) {
+            for (int i = 0; i < data.length; i++) {
```

Comment:
```suggestion
        for (String name : names) {
            int salary = 0;
            for (int i = 0; i < data.length; i++) {
```
this way you won't need to reset it after each loop